src/main/java/com/xeeva/catalog/RequisitioningPage.java

```diff
@@ -1,763 +1,764 @@
+		btnSeeMore.syncEnabled(5);
+		//btnSeeMore.click();
+		driver.executeJavaScript("arguments[0].click();", btnSeeMore);
+		List<WebElement> getRows = tblMainRecentOrdersGrid;
+		int rowsCount = getRows.size();
+		TestReporter.log("Total rows in RecentOrders Grid table: "+ rowsCount);
+
+		for(int row=1; row<=rowsCount; row++){
+			String getStatus = driver.findElement(By.xpath("//*[@id='gvRecentOdersGrid']/tbody/"
+					+ "tr["+ row +"]/td[8]/span")).getText();
+			if(!getStatus.contains("Canceled By Requester")){
+				driver.setElementTimeout(2);
+				driver.findElement(By.xpath("//*[@id='gvRecentOdersGrid']/tbody"
+						+ "/tr["+ row +"]/td[13]/div/a/i")).jsClick();
+				//Handle Alert if present
+				if(AlertHandler.isAlertPresent(driver, 6)){
+					AlertHandler.handleAlert(driver, 6);
+				}
+				txtComments.syncVisible(5);
+				txtComments.safeSet(comments);
+				btnCommentsSubmit.syncVisible(5);
+				btnCommentsSubmit.click();
+				Sleeper.sleep(5000);
+				String getStatusAfterCancelReqLink = driver.findElement(By.xpath("//*[@id='gvRecentOdersGrid']/tbody/"
+						+ "tr["+ row +"]/td[8]/span")).getText();
+				TestReporter.logStep("Get Status After Cancel Requisition: "+getStatusAfterCancelReqLink);
+				TestReporter.assertTrue(getStatusAfterCancelReqLink.contains("Canceled By Requester"), 
+						"Cancelled the requisition record sucessfully.");
+				break;
+			}
+		}
+	}
+*/
+	/**
+	 * @Summary: Method to get the Cart number from Rejected Orders.
+	 * @author: Praveen Namburi, @version: Created 26-09-2016
+	 * @return getCartValue
+	 */
+	public String getCart_RejectedOrders(){
+		String getCartValue="";
+		getCartValue = tblRecentOrder_CartItem.getText().trim();
+		TestReporter.log("Cart Number: "+getCartValue);
+		return getCartValue;
+	}
+
+	/**
+	 * @Summary: Method to get the RFQ number from Rejected Orders tab.
+	 * @author: Praveen Namburi, @version: Created 26-09-2016
+	 * @return getRFQValue
+	 */
+	public String getRFQ_RejectedOrders(){
+		String getRFQValue="";
+		getRFQValue = tblRecentOrder_RFQNum.getText().trim();
+		TestReporter.log("RFQ Number: "+getRFQValue);
+		return getRFQValue;
+	}
+
+	/**
+	 * @Summary: Method to get the REQ number from Rejected Orders.
+	 * @author: Praveen Namburi, @version: Created 26-09-2016
+	 * @return
+	 */
+	public String getREQ_RejectedOrders(){
+		String getREQValue="";
+		String REQValue = tblRecentOrder_REQNum.getText().trim();
+		TestReporter.log("REQ Number: "+ REQValue);
+		if(REQValue.contains("-")){
+			String strREQValue = REQValue.replace("-", "");
+			getREQValue = strREQValue;
+		}
+		return getREQValue;
+	}
+
+	/**
+	 * @summary: Method to enter search Criteria for Rejected Orders and Verify them.
+	 * @author: Praveen Namburi, @version: Created 26-09-2016
+	 * @param cart, @param RFQ, @param OrderDesc, @param location
+	 */
+	public void enterSearchCriteriaAndVerifyRejectedOrders(String location){
+		driver.manage().timeouts().implicitlyWait(7, TimeUnit.SECONDS);
```

Comment:
Remember - element timeouts only affect when using the commands driver.findBy and driver.findBys, and in turn the isDisplayed() and isEnabled() methods.  So if you are not using these methods, then there is no need to change the elmeent timeout.  
If you do need to change the element timeout, please use the orasi driver method: 
driver.setElementTimeout(7);
and then when you're done, to set back to the default:
driver.setElementTimeout(Constants.ELEMENT_TIMEOUT);